doc/api/deprecations.md

```diff
@@ -0,0 +1,567 @@
+
+<a id="DEP00041"></a>
+### DEP00041: NODE\_REPL\_HISTORY\_FILE environment variable
+
+Type: Documentation-only
+
+The `NODE_REPL_HISTORY_FILE` environment variable has been deprecated.
+
+<a id="DEP00042"></a>
+### DEP00042: tls.CryptoStream
+
+Type: Documentation-only
+
+The [`tls.CryptoStream`][] class has been deprecated. Please use
+[`tls.TLSSocket`][] instead.
+
+<a id="DEP00043"></a>
+### DEP00043: tls.SecurePair
+
+Type: Documentation-only
+
+The [`tls.SecurePair`][] class has been deprecated. Please use
+[`tls.TLSSocket`][] instead.
+
+<a id="DEP00044"></a>
+### DEP00044: util.isArray()
+
+Type: Documentation-only
+
+The [`util.isArray()`][] API has been deprecated. Please use `Array.isArray()`
+instead.
+
+<a id="DEP00045"></a>
+### DEP00045: util.isBoolean()
+
+Type: Documentation-only
+
+The [`util.isBoolean()`][] API has been deprecated.
+
+<a id="DEP00046"></a>
+### DEP00046: util.isBuffer()
+
+Type: Documentation-only
+
+The [`util.isBuffer()`][] API has been deprecated. Please use
+[`Buffer.isBuffer()`][] instead.
+
+<a id="DEP00047"></a>
+### DEP00047: util.isDate()
+
+Type: Documentation-only
+
+The [`util.isDate()`][] API has been deprecated.
+
+<a id="DEP00048"></a>
+### DEP00048: util.isError()
+
+Type: Documentation-only
+
+The [`util.isError()`][] API has been deprecated.
+
+<a id="DEP00049"></a>
+### DEP00049: util.isFunction()
+
+Type: Documentation-only
+
+The [`util.isFunction()`][] API has been deprecated.
+
+<a id="DEP00050"></a>
+### DEP00050: util.isNull()
+
+Type: Documentation-only
+
+The [`util.isNull()`][] API has been deprecated.
+
+<a id="DEP00051"></a>
+### DEP00051: util.isNullOrUndefined()
+
+Type: Documentation-only
+
+The [`util.isNullOrUndefined()`][] API has been deprecated.
+
+<a id="DEP00052"></a>
+### DEP00052: util.isNumber()
+
+Type: Documentation-only
+
+The [`util.isNumber()`][] API has been deprecated.
+
+<a id="DEP00053"></a>
+### DEP00053 util.isObject()
+
+Type: Documentation-only
+
+The [`util.isObject()`][] API has been deprecated.
+
+<a id="DEP00054"></a>
+### DEP00054: util.isPrimitive()
+
+Type: Documentation-only
+
+The [`util.isPrimitive()`][] API has been deprecated.
+
+<a id="DEP00055"></a>
+### DEP00055: util.isRegExp()
+
+Type: Documentation-only
+
+The [`util.isRegExp()`][] API has been deprecated.
+
+<a id="DEP00056"></a>
+### DEP00056: util.isString()
+
+Type: Documentation-only
+
+The [`util.isString()`][] API has been deprecated.
+
+<a id="DEP00057"></a>
+### DEP00057: util.isSymbol()
+
+Type: Documentation-only
+
+The [`util.isSymbol()`][] API has been deprecated.
+
+<a id="DEP00058"></a>
+### DEP00058: util.isUndefined()
+
+Type: Documentation-only
+
+The [`util.isUndefined()`][] API has been deprecated.
+
+<a id="DEP00059"></a>
+### DEP00059: util.log()
+
+Type: Documentation-only
+
+The [`util.log()`][] API has been deprecated.
+
+<a id="DEP00060"></a>
+### DEP00060: util.\_extend()
+
+Type: Documentation-only
+
+The [`util._extend()`][] API has been deprecated.
```

Comment:
This (and other) deprecation descriptions should probably be more explicit in terms of mentioning the fact that «deprecated» API in question was never documented to begin with. The documentation for `util._extend` first appeared in https://github.com/nodejs/node/pull/4903, in a PR that deprecated it.

Perhaps something «Was never part of the documented public API» or something like that will do here and in other places where applicable.